source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -85,24 +94,33 @@ def filter_journeys(responses, instance, request):
     composed_filter = ComposedFilter()
     [composed_filter.add_filter(f) for f in filters]
 
-    return composed_filter.compose_filters()(get_qualified_journeys(responses))
+    if is_debug:
+        return composed_filter.compose_filters()(get_all_journeys(responses))
```

Comment:
`compose_filters()` returns a function, so you can declare a variable `f = composed_filter.compose_filters()` and use it

```
f = composed_filter.compose_filters()
if is_debug:
    f(...)
else:
    f(...)
```